Sources/PortalSwift/Components/PortalWebView.swift

```diff
@@ -177,29 +218,24 @@ public class PortalWebView: UIViewController, WKNavigationDelegate, WKScriptMess
     self.onPageStart?()
   }
 
-  /// Called when the web view finishes loading a page. It injects the Portal script into the web view.
+  /// Called when the web view finishes loading a page.
   /// - Parameters:
   ///   - webView: The WKWebView instance that finished loading.
   ///   - navigation: The navigation information associated with the event.
   public func webView(_: WKWebView, didFinish _: WKNavigation!) {
-    guard let address = portal.address else {
-      print("[PortalWebView] No address found for user. Cannot inject provider into web page.")
-      return
-    }
-
-    let javascript = self.injectPortal(
-      address: address,
-      apiKey: self.portal.apiKey,
-      chainId: String(self.portal.chainId),
-      gatewayConfig: self.portal.gatewayConfig[self.portal.chainId]!,
-      autoApprove: self.portal.autoApprove,
-      enableMpc: true
-    )
-
-    self.evaluateJavascript(javascript, sourceURL: "portal_sign")
```

Comment:
where did this code go for evaluateJavascript?

Reply:
ah this is also not necessary. It's handled by the comment below. The `portal_sign` is confusing. It's just the name of the script being injected